### PR TITLE
Feature/PER-218: add labels to the input elements

### PR DESCRIPTION
**Pull request by tnguyen6106**
- Added labels to input boxes
- Styled with some CSS
- Removed header tags that were used in place of the labels previously
- Wrote tests

### CODE REVIEW

File: src/client/src/views/CaptionForm.js [hunk 1]
```diff
@@ -22,20 +22,26 @@ module.exports = {
...
+          id: 'new-name-input',
           oninput: function (e) {
             Scene.currentCaption.name = e.target.value
           },
             value: Scene.currentCaption.name ? Scene.currentCaption.name : ''
         }),
-        m('h2', `Text`),
+        m('label', {
+          for: 'new-caption-text-input'
+        }, 'Text input'),
```
Comment (JosephJindrich) [inline src/client/src/views/CaptionForm.js]: I would call this Caption string, or caption text, not text input

File: src/client/src/views/SceneForm.js [hunk 2]
```diff
@@ -23,20 +23,26 @@ module.exports = {
...
+          id: 'new-name-input',
           oninput: function (e) {
             Scene.currentScene.name = e.target.value
           },
           value: Scene.currentScene.name ? Scene.currentScene.name : ''
         }),
-        m('h2', `Start`),
+        m('label', {
+          for: 'new-start-input'
+        }, 'Start unit'),
```
Comment (JosephJindrich) [inline src/client/src/views/SceneForm.js]: I would call this just Start, instead of start unit

File: src/client/src/views/Start.js [hunk 3]
```diff
@@ -8,7 +8,11 @@ module.exports = {
   view: function() {
     return m('.start', [
       m('h1', 'Welcome!!'),
+      m('label', {
+        for: 'language-input'
+      }, 'Caption specification'),
```
Comment (JosephJindrich) [inline src/client/src/views/Start.js]: This should be Caption Format instead of Caption specification

**Review by JosephJindrich — CHANGES_REQUESTED** [review]
I left some naming changes in the comments, but I'd also like you to change the look a little. I think having the label on top is a good idea, but when there's test in the box that looks identical to the text right above it, it's super off putting. Could you perhaps bold the labels? And maybe make the font 1 to 2 sizes bigger? Normally I wouldn't care about looks, but it's really throwing me for a loop when I look at it. Maybe @evanhackett would like to take a look and see if I'm off about that.

**Comment by evanhackett** [comment]
I agree with @JosephJindrich about changing the wording.

As far as the look and feel, at least for me in my browser, the font of the label is slightly different from the input text, so to me they are distinguished a bit. So I don't necessarily agree with Joe that it looks super off-putting.

However, I see nothing wrong with improving the look. It wouldn't be hard to make the font size bigger, bold it, etc.

So @tnguyen6106 I would just play around with the CSS a bit until you get something that looks somewhat nice.

**Review by JosephJindrich — APPROVED** [review]
Looks great

**Pull request merged**